db_sql.go

```diff
@@ -166,20 +179,27 @@ func (x *sqlUserStoreDB) archivePassword(tx *sql.Tx, userId UserId) error {
 }
 
 func (x *sqlUserStoreDB) hasPasswordBeenUsedBefore(userId UserId, password string) bool {
-	archivedPasswords, err := x.getArchivedPasswordsForUser(userId)
+	pastPasswords, err := x.getPastPasswordsForUser(userId)
```

Comment:
This would read a lot easier if it was `recentPasswords` and `getRecentPasswordsForUser`, instead of `pastpasswords`